src/controller/python/chip/clusters/attribute.cpp

```diff
@@ -426,10 +442,15 @@ chip::ChipError::StorageType pychip_ReadClient_ReadEvents(void * appContext, Dev
         {
             params.mMinIntervalFloorSeconds   = pyParams.minInterval;
             params.mMaxIntervalCeilingSeconds = pyParams.maxInterval;
+            err                               = readClient->SendAutoResubscribeRequest(std::move(params));
+            SuccessOrExit(err);
+            readPaths.release();
```

Comment:
The `release()` here needs to be before the `SuccessOrExit`.  Ideally before the `SendAutoResubscribeRequest`, so it can't get a `SuccessOrExit` inserted before it...

Reply:
yep, forget to move, thanks